devsetup/roles/download_tools/defaults/main.yaml

```diff
@@ -14,3 +14,6 @@ go_version: 1.19.5
 
 # kustomize version to use (must be specific version)
 kustomize_version: v4.5.7
+
+# kustomize version to use (must be specific version)
+kubectl_version: v1.27.3
```

Comment:
crc is using a 1.25 build of kubernetes:

`Kubernetes Version: v1.25.7+eab9cc9
`

kubectl needs to be at most v.1.26 according to https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#before-you-begin.

Reply:
Done